miniapp/src/main/java/com/rakuten/mobile/miniapp/display/Displayer.kt

```diff
@@ -0,0 +1,4 @@
+package com.rakuten.mobile.miniapp.display
+
+@SuppressWarnings("UseDataClass")
+internal class Displayer(val realMiniAppView: RealMiniAppView)
```

Comment:
I'm not sure if this should be passed in the constructor? At the very least, it should be passing the interface, not the implementation.

- [ ] Fix Displayer constructor